worlds/shivers/Rules.py

```diff
@@ -123,7 +123,8 @@ def get_rules_lookup(player: int):
             "To Burial From Egypt": lambda state: state.can_reach("Egypt", "Region", player),
             "To Gods Room From Anansi": lambda state: state.can_reach("Gods Room", "Region", player),
             "To Slide Room": lambda state: all_skull_dials_available(state, player),
-            "To Lobby From Slide Room": lambda state: (beths_body_available(state, player))
+            "To Lobby From Slide Room": lambda state: (beths_body_available(state, player)),
+            "To Water Capture From Janitor Closet": lambda state: (cloth_capturable(state, player))
```

Comment:
Why the parentheses? :O

*low priority*

Reply:
Fixed